.github/workflows/update-ref-tests.yml

```diff
@@ -64,29 +89,47 @@ jobs:
             - [ ] All tests must pass
 
       - name: Create Asana task when workflow succeeded
-        if: ${{ steps.update-check.outcome == 'failure' }}
-        uses: honeycombio/gha-create-asana-task@main
+        id: create-task
+        uses: malmstein/github-asana-action@master
         with:
-          asana-secret: ${{ secrets.GH_ASANA_SECRET }}
-          asana-workspace-id: ${{ secrets.GH_ASANA_WORKSPACE_ID }}
-          asana-project-id: ${{ secrets.GH_ASANA_AOR_PROJECT_ID }}
-          asana-section-id: ${{ secrets.GH_ASANA_INCOMING_ID }}
-          asana-task-name: Update reference tests to latest version
+          asana-pat: ${{ secrets.GH_ASANA_SECRET }}
+          asana-project: ${{ secrets.GH_ASANA_AOR_PROJECT_ID }}
+          asana-section: ${{ secrets.GH_ASANA_INCOMING_ID }}
+          asana-task-name: Update content scope scripts to version ${{ env.VERSION }}
           asana-task-description: |
             Reference tests have been updated and a PR created.
             
             If tests failed check out https://app.asana.com/0/0/1203766026095653/f for further information on what to do next.
             
             See ${{ steps.create-pr.outputs.pull-request-url }}
+          action: 'create-asana-task'
+
+      - name: Add Asana task to Release Board project when workflow succeeded
+        uses: malmstein/github-asana-action@master
+        with:
+          asana-pat: ${{ secrets.GH_ASANA_SECRET }}
+          asana-project: '488551667048375'
```

Comment:
All these should be a secret

Reply:
agreed to move use vars instead of secrets, we don’t need to encrypt these